showcase/app/templates/components/app-header.hbs

```diff
@@ -0,0 +1,175 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "AppHeader Component"}}
+
+<Shw::Text::H1>AppHeader</Shw::Text::H1>
+
+<section data-test-percy>
+  <Shw::Text::H2>Content</Shw::Text::H2>
+
+  <Shw::Text::H3>With generic content</Shw::Text::H3>
+
+  <Shw::Grid @columns={{1}} {{style gap="2rem"}} as |SG|>
+    <SG.Item>
+      <Hds::AppHeader>
+        <:globalBefore>
+          <Shw::Placeholder @height="2em" @width="auto" @text="ProductMenu" />
+        </:globalBefore>
+
+        <:logo>
+          <Shw::Placeholder @height="2em" @width="auto" @text="LOGO" />
+        </:logo>
+
+        <:globalAfter>
+          <Shw::Placeholder @height="2em" @width="auto" @text="OrgPicker" />
+        </:globalAfter>
+
+        <:utility>
+          <Shw::Placeholder @height="2em" @width="auto" @text="Search" />
+          <Shw::Placeholder @height="2em" @width="auto" @text="HelpMenu" />
+          <Shw::Placeholder @height="2em" @width="auto" @text="GeoPicker" />
+          <Shw::Placeholder @height="2em" @width="auto" @text="UserMenu" />
+        </:utility>
+      </Hds::AppHeader>
+    </SG.Item>
+  </Shw::Grid>
+
+  <Shw::Divider @level="2" />
+
+  <Shw::Text::H3>With child component content</Shw::Text::H3>
+
+  <Shw::Grid @columns={{1}} {{style gap="2rem"}} as |SG|>
+    <SG.Item @label="With minimum recommended phase 1 content">
+      <Hds::AppHeader>
+        <:logo>
+          <Hds::AppHeader::HomeLink @icon="hashicorp" @ariaLabel="HashiCorp home menu" @href="#" />
+        </:logo>
+
+        <:globalAfter>
+          <Hds::Dropdown class="hds-side-nav__dropdown" as |dd|>
```

Comment:
**Note:** classnames are used here _temporarily_ for initial rough visual styling purposes until visual designs are more nailed down and also until we decide upon the best way to handle dark theming.